src/cryptoadvance/specter/templates/settings/general_settings.jinja

```diff
@@ -23,7 +23,9 @@
 				</p>
 			</div>
 			{{ _("Specter Data Backup") }}:
-			<br><br>
+			<div class="note">
+			    {{ _("Warning: This backup does not include private keys of the hot wallets") }}
```

Comment:
```suggestion
			    {{ _("Warning: This backup does not include private seed of the hot wallets or, obviously, the private seed of your Hardwarewallets.") }}
```
It's pretty obvious but maybe worth mentioning nevertheless?

Reply:
Yeah we may mention that, will be helpful to new users.